pnglib.js

```diff
@@ -204,4 +204,9 @@
 		}
 	}
 
+  if (typeof module !== 'undefined' && typeof module.exports !== 'undefined') {
```

Comment:
I wish I could think of a way around modifying Robert's source, but I think we have to. Looks like he was using hard-tabs. Would you be ok with matching the tab style and adding a quick comment above each of the modifications to indicate they are changes from the original source?